Userland/Applications/Terminal/main.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2018-2021, Andreas Kling <kling@serenityos.org>
```

Comment:
I don't recall making this change :^)

You can add a line with your own copyright instead, if you like. But please don't change other people's lines.

Reply:
ok, done.